### ISSUE TITLE
would you please add a checkbox for show unique which for filtering data on log page(domain) column 

### ISSUE THREAD

**Issue by kenkilaw** [AdguardTeam/AdGuardHome#733]
would you please add a checkbox for show unique which for filtering data on log page(domain) column? Thanks a lot

**Comment by ameshkov** [comment]
Uh, I don't quite understand what do you want us to do. Could you please supply a little bit more details?

**Comment by kenkilaw** [comment]
sorry for that, I means when i searching a domain which wrongly blocked on adgurad home logging page.
even i filter with blocked domain and host ip, it is showing many record with same domain.
I just think, if there have a function can group/filter same domain as one will be perfect.
or any suggestion that i can be easily to search which domain are wrongly blocked??

btw, as you said Adguard home almost perfect, thanks you guys doing great job, Thanks

**Comment by ameshkov** [comment]
Hmm, that'd be hard to change without breaking other user scenarios.

If you have any examples of this done as you like it, please post them here.

> btw, as you said Adguard home almost perfect, thanks you guys doing great job, Thanks

Thank you!:)

**Comment by ArchiveBase** [comment]
"domain and host ip, it is showing many record with same domain."
   because its each query with time. it's important to us sometimes. If you click 'unblock' that one domain then AGH no longer block it

" if there have a function can group/filter same domain as one will be perfect."
    Top blocked domains on dashboard already as you expect. Maybe i'm wrong? (it doesn't show all but if it extend might be ok for you)

**Comment by ameshkov** [comment]
Merged into #733
